Samples/src/IntroToIQSharp/Notebook.ipynb

```diff
@@ -56,138 +90,425 @@
+      "text/html": [
+       "<ul><li>HelloQuantum</li></ul>"
+      ],
+      "text/plain": [
+       "HelloQuantum"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "operation InvalidQ() : Unit {\n",
-    "\n",
-    "    // The `FooBar` operation doesn't exist, so the following line\n",
-    "    // will generate a `No variable with that name exists.` error:\n",
-    "    FooBar(\"Hello again!\");\n",
-    "    \n",
-    "    // `Message` takes only one string argument, so the following line\n",
-    "    // will generate a `Unexpected argument tuple.` error:\n",
-    "    Message(1, 2);\n",
+    "operation HelloQuantum() : Result {\n",
+    "    using(q = Qubit()) {\n",
+    "        return M(q);\n",
+    "    }\n",
     "}"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "slideshow": {
+     "slide_type": "notes"
+    }
+   },
    "source": [
-    "Q# operations can call other operations previously defined; they can also call all the operations defined in the \n",
-    "[Microsoft.Quantum.Intrinsic](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic) and \n",
-    "[Microsoft.Quantum.Canon](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.canon) namespaces (like [Message](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.message)).\n",
-    "\n",
-    "For example, you can create a new operation that calls the previously compiled `HelloQ`:\n"
+    "To verify this, you can use the `%simulate` magic to simulate the execution of the operation in a QPU:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 3,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "-"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "0",
+      "text/plain": [
+       "Zero"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate HelloQuantum"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "notes"
+    }
+   },
+   "source": [
+    "A more interesting example for a Q# operation is a quantum random number generator (QNRG). A QNRG returns a different value each time it is invoked:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "[\"QNRG\"]",
+      "text/html": [
+       "<ul><li>QNRG</li></ul>"
+      ],
+      "text/plain": [
+       "QNRG"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Diagnostics;\n",
+    "    \n",
+    "operation QNRG() : Result {\n",
```

Comment:
Similarly here, I'd suggest `SampleQrng` as a name to emphasize that the name represents an operation, and following .NET-style acronym abbreviations.